composer.json

```diff
@@ -12,16 +12,16 @@
         }
     ],
     "require": {
-        "php": "^7.1.3",
```

Comment:
Why did you changed PHP version?

Reply:
I thought it wouldn't work with PHP < 7.2. I've changed it back.